tests/unit/s2n_security_policies_test.c

```diff
@@ -816,9 +816,9 @@ int main(int argc, char **argv)
         if (s2n_is_tls13_fully_supported()) {
             /* 20250211 */
             {
-                EXPECT_OK(s2n_test_security_policies_compatible(&security_policy_rfc9151, "default_tls13", ecdsa_sha384_chain_and_key));
-                EXPECT_OK(s2n_test_security_policies_compatible(&security_policy_rfc9151, "default_fips", ecdsa_sha384_chain_and_key));
-                EXPECT_OK(s2n_test_security_policies_compatible(&security_policy_rfc9151, "20250211", ecdsa_sha384_chain_and_key));
+                EXPECT_OK(s2n_test_security_policies_compatible(&security_policy_20250429, "default_tls13", ecdsa_sha384_chain_and_key));
```

Comment:
Did this test actually want a non-fixed RFC9151? Same question for below.

Reply:
Are you suggesting we add that framework here but with a single element in the list? I guess that seems doable, though I'd be surprised if anyone notices that it needs updating if there's a new rfc9151 fixed version?

Happy to add it if that's what you mean.

Reply:
Ah, it looks like we do in fact have that check :)

https://github.com/aws/s2n-tls/blob/10e684339305e65e9a301c09816fca002031a900/tests/unit/s2n_security_policies_test.c#L76